includes/grids-toolbar.md

```diff
@@ -0,0 +1,53 @@
+  </tr>
+  <tr>
+    <td><i>applyFilterButton</i></td>
+    <td><img src="/images/DataGrid/icons/toolbar_applyFilter.png" alt="DevExtreme JavaScript jQuery Angular UI component {WidgetName} Toolbar ApplyFilterButton" style="margin:1px; vertical-align:middle"/></td>
+    <td>filterRow.<a href="https://js.devexpress.com/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/filterRow/#visible">visible</a> is set to <b>true</b> and filterRow.<a href="https://js.devexpress.com/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/filterRow/#applyFilter">applyFilter</a> is set to <i>"onClick"</i></td>
+  </tr>
+  <tr>
+    <td><i>columnChooserButton</i></td>
+    <td><img src="/images/DataGrid/icons/columnChooser.png" alt="DevExtreme JavaScript jQuery Angular UI component {WidgetName} Toolbar ColumnChooserButton" style="margin:1px; vertical-align:middle"/></td>
+    <td>columnChooser.<a href="https://js.devexpress.com/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/columnChooser/#enabled">enabled</a> is set to <b>true</b></td>
+  </tr>
+  <tr>
+    <td><i>revertButton</i></td>
+    <td><img src="/images/DataGrid/icons/toolbar_revert.png" alt="DevExtreme JavaScript jQuery Angular UI component {WidgetName} Toolbar RevertButton" style="margin:1px; vertical-align:middle"/></td>
+    <td><a href="https://js.devexpress.com/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/editing/">editing</a>.<a href="https://js.devexpress.com/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/editing/#mode">mode</a> is set to <i>"batch"</i></td>
+  </tr>
+  <tr>
+    <td><i>saveButton</i></td>
+    <td><img src="/images/DataGrid/icons/toolbar_save.png" alt="DevExtreme JavaScript jQuery Angular UI component {WidgetName} Toolbar SaveButton" style="margin:1px; vertical-align:middle"/></td>
+    <td><a href="https://js.devexpress.com/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/editing/">editing</a>.<a href="https://js.devexpress.com/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/editing/#mode">mode</a> is set to <i>"batch"</i> and <a href="https://js.devexpress.com/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/editing/">editing</a>.<a href="https://js.devexpress.com/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/editing/#allowUpdating">allowUpdating</a> is set to <b>true</b></td>
+  </tr>
+  <tr>
+    <td><i>searchPanel</i></td>
+    <td><img src="/images/DataGrid/icons/toolbar_searchPanel.png" alt="DevExtreme JavaScript jQuery Angular UI component {WidgetName} Toolbar searchPanel" style="margin:1px; vertical-align:middle"/></td>
```

Comment:
The search panel lacks space. Let's make it a bit wider and match the width with the new group panel illustration.